.github/workflows/cicd.yaml

```diff
@@ -4,155 +4,75 @@ on:
   push:
     branches: ["ise"]
   workflow_dispatch:
-  
-env:
-  DOCKER_PREFIX: ghcr.io/${{ github.repository }}
-  DOCKER_TAG: ${{ vars.VERSION }}-l1vh.${{ github.run_number }}
-  CUSTOM_LIBVIRT_IMAGE: ${{ format('ghcr.io/{0}/libvirt-rpms:{1}', github.repository, 'qemu-mshv') }}
-  CUSTOM_QEMU_IMAGE: ${{ format('ghcr.io/{0}/qemu-rpms:{1}', github.repository, 'qemu-mshv') }}
+    inputs:
```

Comment:
The problem with this approach is that when the workflow is invoked in a regular way (push to ISE) all these inputs will be empty (see https://github.com/kaizentm/kubevirt/actions/runs/18144985839). In CI/CD I would't introduce any inputs (why?) but just leave them as it was in ENV or we can move them/some to the repo variables if we really want. But we can have the inputs in "Build and Push Images" workflows and we can add workflow_dispatch triggering event (duplicating all the inputs) so that we can run it manually when we just need to rebuild images with some custom parameters (registry, tag, rpm, etc.)